akka-stream/src/main/scala/akka/stream/impl/PhasedFusingActorMaterializer.scala

```diff
@@ -779,7 +779,7 @@ final class TlsModulePhase(settings: ActorMaterializerSettings, materializer: Ph
     publishers(out.id)
 
   def takePublisher(slot: Int, publisher: Publisher[Any]): Unit =
-    publisher.subscribe(FanIn.SubInput[Any](tlsActor, slot))
+    publisher.subscribe(FanIn.SubInput[Any](tlsActor, 1-slot))
```

Comment:
It might be the ordering here (in `TlsModule`):

```
val shape = new BidiShape(plainIn, cipherOut, cipherIn, plainOut)
```

vs. the numbering of the boundaries in `TLSActor`:

```
  final val TransportIn = 0
  final val TransportOut = 0

  final val UserOut = 1
  final val UserIn = 1
```